src/status_im/i18n.cljs

```diff
@@ -65,65 +65,66 @@
-                             :fi          fi/translations
-                             :fr          fr/translations
-                             :fr-ch       fr-ch/translations
-                             :fy          fy/translations
-                             :he          he/translations
-                             :hi          hi/translations
-                             :hu          hu/translations
-                             :id          id/translations
-                             :it          it/translations
-                             :it-ch       it-ch/translations
-                             :ja          ja/translations
-                             :ko          ko/translations
-                             :la          la/translations
-                             :lt          lt/translations
-                             :lv          lv/translations
-                             :ms          ms/translations
-                             :nb          nb/translations
-                             :ne          ne/translations
-                             :nl          nl/translations
-                             :pl          pl/translations
-                             :pt-br       pt-br/translations
-                             :pt-pt       pt-pt/translations
-                             :ro          ro/translations
-                             :ru          ru/translations
-                             :sl          sl/translations
-                             :sr          sr-rs-cyrl/translations
-                             :sr-RS_#Latn sr-rs-latn/translations
-                             :sr-RS_#Cyrl sr-rs-cyrl/translations
-                             :sv          sv/translations
-                             :sw          sw/translations
-                             :th          th/translations
-                             :tr          tr/translations
-                             :uk          uk/translations
-                             :ur          ur/translations
-                             :vi          vi/translations
-                             :zh          zh-hans/translations
-                             :zh-hans     zh-hans/translations
-                             :zh-hans-cn  zh-hans/translations
-                             :zh-hans-mo  zh-hans/translations
-                             :zh-hant     zh-hant/translations
-                             :zh-hant-tw  zh-hant-tw/translations
-                             :zh-hant-sg  zh-hant-sg/translations
-                             :zh-hant-hk  zh-hant-hk/translations
-                             :zh-hant-cn  zh-hant/translations
-                             :zh-hant-mo  zh-hant/translations
-                             :zh-wuu      zh-wuu/translations
-                             :zh-yue      zh-yue/translations})
+(def translations-by-locale {:en          en/translations}) ; Temporarily disable all languages except English
```

Comment:
This style of comments is pretty intrusive. Please rather use `#_` before `translations-by-locale` form.

Reply:
Will do, thanks for the tips!